blue_green_deploy.go

```diff
@@ -48,8 +55,39 @@ func (p *BlueGreenDeploy) DeleteAllAppsExceptLiveApp(appName string) {
 
 }
 
-func (p *BlueGreenDeploy) PushNewApp(appName string, route plugin_models.GetApp_RouteSummary, manifestPath string) {
+func (p *BlueGreenDeploy) GetScaleParameters(appName string) AppScaleParameters {
+	appModel, err := p.Connection.GetApp(appName)
+	if err != nil {
+		return AppScaleParameters{}
```

Comment:
I think we should handle this error properly - return this error so the user knows what went wrong so change the function to return an error.